apps/point-of-sale/src/components/Cart/CartActionsComponent.vue

```diff
@@ -73,14 +84,15 @@ const finalizeCheckout = async () => {
     sound.value = new Audio('sounds/rct-cash.wav');
     sound.value.play();
   }
+
   await cartStore.checkout();
   checkingOut.value = false;
   duration.value = 3;
   await logoutService();
 };
 const checkout = async () => {
   if (!enabled.value) return;
-
+  if (stopCheckoutForUserInDebt.value) return stopCheckout();
```

Comment:
Note that there is still an automatic checkout after 30 seconds.

If you have an order which you cannot place because you would go into debt, the automatic checkout will still try and place the order, which will fail and the session wont close.